### PR TITLE
(GH-52) README with PCT Install & Quick Start guide

### PR DESCRIPTION
**Pull request by sanfrancrisko**
Reviewable with GIFS [here](https://github.com/puppetlabs/pdkgo/blob/35dbd64e45aed63ae7f2166c9a9b1eb45eda51a0/README.md)

Closes: #52

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,19 +22,69 @@ These are meant to be ready-to-run, which means they put everything needed for a
 
 > :warning: PCT is currently in an EXPERIMENTAL phase and feedback is encouraged via [pdkgo/discussions](https://github.com/puppetlabs/pdkgo/discussions) and starting a `feedback` post.
 
-## Getting Started
-
-Grab the `experimental` PCT release available from [github.com/puppetlabs/pdkgo/releases/](https://github.com/puppetlabs/pdkgo/releases/).
+## Installing
```
Comment (jpogran) [inline README.md:25]: These new headers need to be added to the TOC at the begining

File: README.md [hunk 2]
```diff
@@ -22,19 +22,69 @@ These are meant to be ready-to-run, which means they put everything needed for a
...
 
-Uncompress the archive to a location of your choosing - this will be refered to as `$INSTALLATION_ROOT` subsequently.
+Use the `install.[ps1|sh]` script, depending upon your OS:
```
Comment (jpogran) [inline README.md:27]:
```suggestion
While the PCT is in early release, we provide an archive and a simple script to unpack it. When we move closer to a full release we will add a platform specific installer. Use the `install.[ps1|sh]` script, depending upon your OS:
```

**Review by jpogran — CHANGES_REQUESTED** [review]

**Review by da-ar — CHANGES_REQUESTED** [review]
Change the commit from "draft of". 
In future avoid WIP or Draft in the commit and just use a draft PR to signify work in progress.

Apart from that it looks great, nice work

**Comment by sanfrancrisko** [comment]
> Change the commit from "draft of".
> In future avoid WIP or Draft in the commit and just use a draft PR to signify work in progress.
> 
> Apart from that it looks great, nice work

Good catch - forgot to rename that in the rebase

**Review by da-ar — APPROVED** [review]

**Review by jpogran — APPROVED** [review]

**Pull request merged**